packages/liveblocks-react/src/ThreadDB.ts

```diff
@@ -178,20 +179,23 @@ export class ThreadDB<M extends BaseMetadata> {
    *   'desc' means by updatedAt DESC
    *
    * Will never return deleted threads in the result.
+   *
+   * Subscriptions are needed to filter threads based on the user's subscriptions.
    */
   public findMany(
     // TODO: Implement caching here
     roomId: string | undefined,
     query: ThreadsQuery<M> | undefined,
-    direction: "asc" | "desc"
+    direction: "asc" | "desc",
+    subscriptions: SubscriptionsByKey | undefined
```

Comment:
What's your opinion on making this optional instead of asking `undefined`?

Reply:
I prefer to have `undefined` so that it forces you to check if you need to pass it or not. But I don't have a strong opinion on it.